raven/src/main/java/com/getsentry/raven/event/Event.java

```diff
@@ -275,7 +275,7 @@ public String toString() {
     /**
      * Levels of log available in Sentry.
      */
-    public static enum Level {
+    public enum Level {
```

Comment:
Also, for posterity: the `static` keyword was removed here due to a checkstyle violation since `enum` types are implicitly static.